announcement/models/res_users.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2022 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+
+class ResUsers(models.Model):
+    _inherit = "res.users"
+
+    pendant_announcement_ids = fields.Many2many(
+        comodel_name="announcement", relation="pendant_announcement_user_rel",
+    )
+    read_announcement_ids = fields.Many2many(
+        comodel_name="announcement", relation="read_announcement_user_rel",
+    )
+    popup_announcements = fields.Boolean()
```

Comment:
`store=False` ?

Reply:
It's not a computed field. Would it be stored in cache?

Reply:
Yes, it is

Reply:
I just tried and it doesn't seem to work for this purpose

Reply:
OK